tests/components/matter/snapshots/test_sensor.ambr

```diff
@@ -2523,120 +2523,6 @@
     'state': '0.0',
   })
 # ---
-# name: test_sensors[silabs_dishwasher][sensor.dishwasher_current-entry]
```

Comment:
So why does this change remove dishwasher entities?

Reply:
that is indeed strange! Maybe the fixture got overwritten in a previously merged PR ?

Reply:
But then the test would fail

Reply:
I just tried on `dev` and it just passed, so this PR is doing something to this

Reply:
found the issue in the discovery logic - I will fix that first in a dedicated PR and then rebase this one